Resources/Locale/en-US/deltav/markings/vulpkanin.ftl

```diff
@@ -102,14 +102,6 @@ marking-VulpTailTip-vulp = Vulpkanin tail (base)
 marking-VulpTailTip-vulp-tip = Vulpkanin tail (tip)
 marking-VulpTailTip = Vulpkanin (tip)
 
-marking-VulpTailWag-vulp_wag = Vulpkanin tail (base)
```

Comment:
This removes a lot of existing markings, but does not provide a db migration. This can lead to unexpected errors at runtime.

Reply:
It shouldn't